lib/lita/handlers/stats.rb

```diff
@@ -0,0 +1,52 @@
+require 'signalfx'
+
+module Stats
+  module_function
+  SIGNALFX_TOTEMS_METRICS_DASHBOARD = "https://app.signalfx.com/#/dashboard/FNSkEUVAYAA"
+
+  def signalfx_client
+    @signalfx_client ||= SignalFx.new ENV['LITA_SIGNALFX_TOKEN']
+  end
+
+  def send_to_signalFX(metric, value)
+    signalfx_client.send( 
+      gauges:
+        [ 
+          {  
+          :metric => metric,
+          :value => value,
+          :timestamp => (Time.now.to_f * 1000).to_i
+        }
+      ]
+    )
+  end
+
+  # captures how many times a totem has being added
+  def capture_totem_use(totem)
+    send_to_signalFX("totems:add:#{totem}", 1)
+  end
+
+  # captures how many people are waiting for a totem.
+  def capture_people_waiting(totem)
+    send_to_signalFX("totems:people_waiting:#{totem}", 1)
```

Comment:
If this is a gauge should it be more than 1 sometimes (the current number of people waiting). Is it meant to be a counter instead?